project/Dependencies.scala

```diff
@@ -181,7 +187,7 @@ object Dependencies {
         Provided.activation // dockerClient needs javax.activation.DataSource in JDK 11+
       )
 
-  val actorTestkitTyped = l ++= Seq(Provided.junit, Provided.scalatest.value)
+  val actorTestkitTyped = l ++= Seq(Provided.logback, Provided.junit, Provided.scalatest.value)
```

Comment:
Since the testkit actually uses logback, shouldn't it be
```suggestion
  val actorTestkitTyped = l ++= Seq(Compile.logback, Provided.junit, Provided.scalatest.value)
```
?

Reply:
No, don't want to include it as transitive dependency. It's only the logging filter features that requires Logback. Same relation as for junit and scalatest.

This translates to the following in the pom.xml:
```
        <dependency>
            <groupId>ch.qos.logback</groupId>
            <artifactId>logback-classic</artifactId>
            <version>1.2.3</version>
            <scope>test</scope>
            <optional>true</optional>
        </dependency>
        <dependency>
            <groupId>junit</groupId>
            <artifactId>junit</artifactId>
            <version>4.12</version>
            <scope>test</scope>
            <optional>true</optional>
        </dependency>
```